sound/soc/sof/compress.c

```diff
@@ -11,6 +11,27 @@
 #include "sof-audio.h"
 #include "sof-priv.h"
 
+struct sof_compr_stream {
+	unsigned int copied_total;
+	unsigned int sample_rate;
+};
+
+static void sof_set_transferred_bytes(struct sof_compr_stream *sstream,
+				      u64 host_pos, uint64_t buffer_size)
```

Comment:
`u64` instead of `uint64_t`

Reply:
@lyakh actually this was my first intention, but checkpatch.pl says u64 kernel type should be used. Let me go back and read more about this!

Reply:
`snd_compr_runtime.buffer_size` is u64

Reply:
While at it I think buffer_size should also be `u64`.

Reply:
"`u64` instead of `uint64_t`" - that's exactly what I meant, yes, the type of `buffer_size`, sorry for being unclear.